### PR TITLE
[WIP] Add Motr to rgw-admin

### PR DESCRIPTION
**Pull request by zalsader**
Attempting to change the script `radosgw-admin` to make the store configurable.
To test this, run: 

```shell
cd /your/ceph-build
bin/radosgw-admin user info --uid=tester
```
It should return some information about the default user.

### CODE REVIEW

**Comment by zalsader** [comment]
Does not work yet, gives the following error:
```
[root@centos8 build]# bin/radosgw-admin user info --uid=tester
2021-12-09T05:31:28.774+0000 7ff5da8c7440 -1 WARNING: all dangerous and experimental features are enabled.
2021-12-09T05:31:28.806+0000 7ff5da8c7440 -1 WARNING: all dangerous and experimental features are enabled.
2021-12-09T05:31:28.809+0000 7ff5da8c7440 -1 WARNING: all dangerous and experimental features are enabled.
2021-12-09T05:31:28.811+0000 7ff5da8c7440  1 RGW CONF BACKEND STORE = motr
2021-12-09T05:31:28.811+0000 7ff5da8c7440  1 RGW BACKEND STORE = motr
2021-12-09T05:31:28.811+0000 7ff5da8c7440  0 newMotrStore()
2021-12-09T05:31:28.812+0000 7ff5da8c7440  0 INFO: my motr endpoint:  192.168.64.130@tcp:12345:4:1
2021-12-09T05:31:28.812+0000 7ff5da8c7440  0 INFO: ha agent endpoint: 192.168.64.130@tcp:12345:1:1
2021-12-09T05:31:28.812+0000 7ff5da8c7440  0 INFO: my motr fid:       0x7200000000000001:0x29
2021-12-09T05:31:28.812+0000 7ff5da8c7440  0 INFO: motr profile fid:  0x7000000000000001:0x4f
motr[1063094]:    50  ERROR  [/root/rpmbuild/BUILD/cortx-motr-2.0.0/net/lnet/ulnet_core.c:1153:nlx_core_tm_start]  <! rc=-98 The address 0x20000c0a84082:12345:4:1 is already in use by another process.
motr[1063094]:  8980  ERROR  [rpc/rpc_machine.c:509:rpc_tm_setup]  <! rc=-101 TM start
motr[1063094]:  8980  ERROR  [rpc/rpc_machine.c:173:m0_rpc_machine_init]  <! rc=-101
2021-12-09T05:31:29.828+0000 7ff5da8c7440  0 ERROR: m0_client_init() failed: -101
2021-12-09T05:31:29.828+0000 7ff5da8c7440  0 newMotrStore() failed!
couldn't init storage provider
```
Currently investigating, but if anyone got ideas that would be appreciated

File: src/rgw/rgw_admin.cc [hunk 1]
```diff
@@ -3992,10 +3992,29 @@ int main(int argc, const char **argv)
     bool need_cache = readonly_ops_list.find(opt_cmd) == readonly_ops_list.end();
     bool need_gc = (gc_ops_list.find(opt_cmd) != gc_ops_list.end()) && !bypass_gc;
 
+    // TODO DRY
```
Comment (johnbent) [inline src/rgw/rgw_admin.cc:3991]: What does this comment mean?

Comment (zalsader) [reply src/rgw/rgw_admin.cc:3991]: Don't repeat yourself, this code is copy-pasted from another place and could be moved into a function or something.

**Comment by johnbent** [comment]
So please do the appropriate thing and create the shared function.
________________________________

**Pull request closed without merging**